changelogs/fragments/7897-consul-action-group.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - consul_auth_method, consul_binding_rule, consul_policy, consul_role, consul_session, consul_token - Added action group ``consul`` (https://github.com/ansible-collections/community.general/pull/7897).
```

It's better to use the full name, since users otherwise might think they can use `group/consul` instead of `group/community.general.consul` after reading this.
```suggestion
  - consul_auth_method, consul_binding_rule, consul_policy, consul_role, consul_session, consul_token - added action group ``community.general.consul`` (https://github.com/ansible-collections/community.general/pull/7897).
```